src/deriver/tom/embeddings.py

```
@@ -1,117 +1,642 @@
 
-from ... import crud, schemas
+from ... import crud, models, schemas
 from ...dependencies import tracked_db
+from ...utils.history import SummaryType, get_session_summaries
+from ..fact_saver import get_fact_saver_queue
+from src.utils.deriver import format_datetime_simple
 
 logger = logging.getLogger(__name__)
 
 
 class CollectionEmbeddingStore:
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)